sklearn/utils/_tags.py

```diff
@@ -274,19 +286,20 @@ def default_tags(estimator) -> Tags:
     tags : Tags
         The default tags for the estimator.
     """
-    from ..base import is_classifier, is_regressor
-
-    target_required = is_classifier(estimator) or is_regressor(estimator)
+    est_is_classifier = getattr(estimator, "_estimator_type", None) == "classifier"
+    est_is_regressor = getattr(estimator, "_estimator_type", None) == "regressor"
+    target_required = est_is_classifier or est_is_regressor
```

Comment:
Once `_estimator_type` is removed, what will we do here?

Reply:
we'll be removing `default_tags` alltogether.